src/github/index.ts

```diff
@@ -54,4 +55,6 @@ export default (robot: Application) => {
 
 	robot.on("create", middleware(createBranch));
 	robot.on("delete", middleware(deleteBranch));
+
+	robot.on("repository.deleted", middleware(deleteRepository));
```

Comment:
I just tested deleting a repository and the webhook event I saw was `installation_repositories.removed`

Reply:
Yeah you'll need to make your local app listen to the `repository` event now and that should work.